src/App/component/AddRepositoryDialog.jsx

```diff
@@ -65,45 +80,41 @@ export default function AddRepositoryDialog({ open, reloadProjects, handleClose,
   const checkRepositoryURL = async (url) => {
     if (repoType === "github") {
       try {
-        await Axios.get(`http://localhost:9100/pvs-api/repository/github/check?url=${url}`,
-          { headers: { "Authorization": `${jwtToken}` } });
-        return true;
+        await sendPVSBackendRequest('GET', `/repository/github/check?url=${url}`)
+        return true
       } catch (e) {
-        alert("github error");
-        return false;
+        alert("Invalid GitHub URL")
+        return false
       }
     }
 
     if (repoType === "gitlab") {
       try {
-        await Axios.get(`http://localhost:9100/pvs-api/repository/gitlab/check?url=${url}`,
-          { headers: { "Authorization": `${jwtToken}` } });
-        return true;
+        await sendPVSBackendRequest('GET', `/repository/gitlab/check?url=${url}`)
```

Comment:
Suggest use `axios` api to send a GET request with query parameters.

So may be you can change the function to this:

```js
  const sendPVSBackendRequest = async (method, url, params) => {
    const baseURL = 'http://localhost:9100/pvs-api'
    const requestConfig = {
      baseURL,
      url,
      method,
      headers,
      params
    }
    return (await Axios.request(requestConfig))?.data
  }
```

and change this line to

```js
await sendPVSBackendRequest('GET', `/repository/gitlab/check`, { url })
```